Sources/Amplitude/Amplitude.m

```diff
@@ -284,6 +280,82 @@ - (instancetype)initWithInstanceName:(NSString *)instanceName {
     return self;
 }
 
+- (NSString *)getDataStorageKey:(NSString *)key {
+    NSString *dataStorageKey = [NSString stringWithFormat:@"%s_%@_%@_%@", "amplitude", _bundleIdentifier ,self.instanceName, key];
+    return dataStorageKey;
+}
+
+- (void)migrateToFileStorage {
+    NSString *fileStoragePath =  [AMPStorage getAppStorageAmpDir:self.instanceName];
+    BOOL isDir;
+    [[NSFileManager defaultManager] fileExistsAtPath:fileStoragePath isDirectory:&isDir];
+    if (isDir) return;
+
+    NSString *databaseDirectory = [AMPUtils platformDataDirectory];
```

Comment:
nit: in theory these two check could be moved to storage and dbhelper as a check method to make it more clear. But since it is only for migration, I guess it is fine to leave it for now.